File: Python/744. Find Smallest Letter Greater Than Target.py
class Solution:
    def nextGreatestLetter(self, letters, target):
        """
        :type letters: List[str]
        :type target: str
        :rtype: str
        """
        if letters[-1] <= target:
            return letters[0]
        else:
            for letter in letters:
                if letter > target:
                    return letter

File: C++/744. Find Smallest Letter Greater Than Target.cpp
class Solution {
public:
    char nextGreatestLetter(vector<char>& letters, char target) {
        if(target >= letters.back())
            return letters[0];
        else
        {
            for(auto letter: letters)
            {
                if(letter > target)
                    return letter;
            }
        }
        
    }
};